classes/Tools.php

```diff
@@ -2836,7 +2836,20 @@ public static function getRobotsContent()
             '*/modules/*.js',
             '*/modules/*.png',
             '*/modules/*.jpg',
-            '/js/jquery/*',
+            '*/modules/*.gif',
+            '*/modules/*.woff',
+            '*/modules/*.woff2',
+            '*/modules/*.ttf'
```

Comment:
```suggestion
            '*/modules/*.ttf',
```

Reply:
@marionf push this suggestion and it is OK to me :)